src/coreclr/jit/target.h

```diff
@@ -5,13 +5,15 @@
 #ifndef TARGET_H_
 #define TARGET_H_
 
-// Native Varargs are not supported on Unix (all architectures) and Windows ARM
-#if defined(TARGET_WINDOWS) && !defined(TARGET_ARM)
-#define FEATURE_VARARG 1
-#else
-#define FEATURE_VARARG 0
+#ifdef TARGET_UNIX_POSSIBLY_SUPPORTED
+#define FEATURE_CFI_SUPPORT
 #endif
 
+// Native Varargs are not supported on Unix (all architectures) and Windows ARM
+#define compFeatureVarArg() (TargetOS::IsWindows && !TargetArchitecture::IsArm32)
+#define compMacOsArm64Abi() (TargetArchitecture::IsArm64 && TargetOS::IsMacOS)
+#define compFeatureArgSplit() ( TargetArchitecture::IsArm32 || (TargetOS::IsWindows && TargetArchitecture::IsArm64) )
+
```

Comment:
Seems odd to have #defines that look like methods; why not just create methods?

Reply:
Turned them into methods. Seems to work fine.